src/Peering/Peering.Test/ScenarioTests/ScenarioExchangePeeringTests.cs

```diff
@@ -57,26 +57,6 @@ public void TestConvertLegacyToExchange()
             TestController.NewInstance.RunPowerShellTest(this.logger, "Test-ConvertLegacyToExchange");
         }
 
-        /// <summary>
-        /// The test update exchange i pv 4 on resource id.
-        /// </summary>
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
@rileymckenna any reason the tests in this file are being removed?

Reply:
These were duplicate tests. or they aren't supported by the back end right now.